doc/command_ref.rst

```diff
@@ -142,8 +142,39 @@ Options
 ``-h, --help``
     Show the help.
 
+.. _installroot-label:
+
 ``--installroot=<path>``
-    set install root
+    Specifies an alternative installroot, relative to where all packages will be
+    installed. Think of this like doing ``chroot <root> dnf`` except using
+    ``--installroot`` allows dnf to work before the chroot is created. The
+    following things are taken from or stored in installroot: cachedir, gpgkey
+    (they are imported into installroot from path described in .repo file that
+    is not relative to installroot), log files, releasever. The following thinks
+    are searched first inside of installroot, and if they are not present, they
+    are taken from host system: config file (unless ``--config=<config file>``
+    command-line option is used, then the path is not relative to installroot),
+    and repository (unless ``--setopt=reposdir=<reposdir>`` command-line option
+    is used, then the path is not relative to installroot).
+    The pluginpath and pluginconfpath are not relative to installroot.
+    Note: You may also want to use the command-line option
+    ``--releasever=<release>`` when creating the installroot otherwise the
+    $releasever value is taken from the rpmdb within the installroot (and thus
+    it is empty at time of creation, the transaction will fail).
+    If installroot is run for first time, repository, releasever, and dnf.conf
+    file are not present in installroot.
```

Comment:
are not present ~~in~~ *within an empty* installroot *directory*. (but still it sounds weird)